jac-uk/admin
# outreach report changes

## What's included?
The Targeted Outreach report page was due a refactor.
Changes in this PR cover:

- "No Candidates or Applications match the given National Insurance Numbers. - Please try again" message when 0 results are returned.
-  'Full screen' functionality
-  Report is horizontally scroll-able.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Navigate to the TA page (candidates > Targeted outreach report)
- Enter a valid NI number 
- Ensure results are shown

- Enter an NI number for which no results should show/an invalid NI number
- Ensure the no results found message is shown

- Try the scrollable report (horizontal)
- Try the fullscreen button

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
